Angular/src/app/models/Fragment.ts

```diff
@@ -39,7 +40,16 @@ export class Fragment {
    * @param fragment with JSON data received from the server
    * @author Ycreak
    */
-  public set_fragment(fragment) {
+  public set_fragment(fragment: any) {
```

Comment:
Fancy logic

Reply:
Angular should have a type called "random json blob from server"